pinot-broker/src/main/java/com/linkedin/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -181,7 +195,7 @@ public BrokerResponse handleRequest(JSONObject request, @Nullable RequesterIdent
 
     // Validate the request
     try {
-      validateRequest(brokerRequest);
+      validateRequest(brokerRequest, request);
```

Comment:
can we parse the json request into a concrete class e.g. RequestParams. This will avoid having constant strings everywhere.